### PR TITLE
sizes, fonts and color fixes

### PR DESCRIPTION
**Pull request by frapperino**

### CODE REVIEW

File: src/pages/Offer/sections/HedvigInfo.tsx [hunk 1]
```diff
@@ -69,15 +68,14 @@ const Row = styled('div')({
   },
 })
 
-const Title = styled('h3')({
+const Title = styled('p')({
```
Comment (palmfjord) [inline src/pages/Offer/sections/HedvigInfo.tsx]: this looks weird to me. Why are we using a title as a `p`?

File: src/pages/Offer/sections/Offer.tsx [hunk 2]
```diff
@@ -156,9 +152,12 @@ export const Offer: React.SFC<Props> = ({ signButtonVisibility, offer }) => (
           <HeaderBackground>
             <HeaderWrapper>
               <Header>
-                <TranslationsConsumer textKey="OFFER_HEADER">
+                <TranslationsPlaceholderConsumer
+                  textKey="OFFER_HEADER"
+                  replacements={{ name: offer.member.firstName }}
```
Comment (palmfjord) [inline src/pages/Offer/sections/Offer.tsx]: could we name this replacement `firstName`? i think that's what we use in other places

File: src/pages/Offer/sections/HedvigInfo.tsx (hunk 1 above)
Comment (frapperino) [reply src/pages/Offer/sections/HedvigInfo.tsx]: It should be a p size wise, maybe title is the wrong naming though

Comment (palmfjord) [reply src/pages/Offer/sections/HedvigInfo.tsx]: i think if it is a title semantically we should just restyle it to be a P

Comment (frapperino) [reply src/pages/Offer/sections/HedvigInfo.tsx]: Hmm, not sure what you mean🤔 there is very little styling done here, so lets leave it as a `p`?

Comment (frapperino) [reply src/pages/Offer/sections/HedvigInfo.tsx]: or did you want a `div`?

File: src/pages/Offer/sections/Offer.tsx [hunk 3]
```diff
@@ -133,7 +129,7 @@ interface Props {
 const COLUMNS = [
   {
     key: 0,
-    title: 'Lagenhetsskydd',
+    title: 'Lägenhetsskydd',
```
Comment (hedvigoscar) [inline src/pages/Offer/sections/Offer.tsx]: We should move this to text keys I think :wink:

File: src/pages/Offer/sections/InsuranceCoverage.tsx [hunk 4]
```diff
@@ -339,6 +345,7 @@ export const InsuranceCoverage: React.SFC = () => (
                     </SwitcherItem>
                   ))}
                 </Switcher>
+                <PerilInfo>(Klicka på ikonerna för mer information)</PerilInfo>
```
Comment (palmfjord) [inline src/pages/Offer/sections/InsuranceCoverage.tsx]: text key?

**Pull request merged**